src/shared/helpers/getCurrentModuleDirectoryPath.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Возвращает путь к каталогу текущего модуля (файла), в котором вызвана эта функция.
+ *
+ * @returns {string} Путь к каталогу текущего модуля.
+ */
+import { fileURLToPath } from 'node:url';
+import { dirname } from 'node:path';
+
+export function getCurrentModuleDirectoryPath() {
+  const filepath = fileURLToPath(import.meta.url);
+  return dirname(filepath);
+}
```

Comment:
Использования JSDoc хорошая практика. 
Комментарий должны быть прямо над функций

```suggestion
import { fileURLToPath } from 'node:url';
import { dirname } from 'node:path';

/**
 * Возвращает путь к каталогу текущего модуля (файла), в котором вызвана эта функция.
 *
 * @returns {string} Путь к каталогу текущего модуля.
 */
export function getCurrentModuleDirectoryPath() {
  const filepath = fileURLToPath(import.meta.url);
  return dirname(filepath);
}
```